lib/framework/NetworkSettingsService.cpp

```diff
@@ -354,31 +354,39 @@ void NetworkSettingsService::WiFiEvent(WiFiEvent_t event, WiFiEventInfo_t info)
             setWiFiPowerOnRSSI();
         }
 #if ESP_IDF_VERSION_MAJOR < 5
-        WiFi.enableIpV6();
+        WiFi.enableIpV6(); // force ipv6
 #endif
         break;
 
     case ARDUINO_EVENT_ETH_CONNECTED:
 #if ESP_IDF_VERSION_MAJOR < 5
-        ETH.enableIpV6();
+        ETH.enableIpV6(); // force ipv6
 #endif
         break;
 
         // IPv6 specific
     case ARDUINO_EVENT_WIFI_STA_GOT_IP6:
-    case ARDUINO_EVENT_ETH_GOT_IP6:
 #if !TASMOTA_SDK && ESP_IDF_VERSION_MAJOR < 5
-        if (emsesp::EMSESP::system_.ethernet_connected()) {
-            emsesp::EMSESP::logger().info("Local IPv6 (Eth) =%s", ETH.localIPv6().toString().c_str());
-        } else {
-            emsesp::EMSESP::logger().info("Local IPv6 (WiFi)=%s", WiFi.localIPv6().toString().c_str());
-        }
+        emsesp::EMSESP::logger().info("Local IPv6 (WiFi)=%s", WiFi.localIPv6().toString().c_str());
 #else
         emsesp::EMSESP::logger().info("Local IPv6=%s", IPAddress(IPv6, (uint8_t *)info.got_ip6.ip6_info.ip.addr, 0).toString().c_str());
 #endif
         emsesp::EMSESP::system_.has_ipv6(true);
         break;
 
+        // IPv6 specific
+        // This a bug in arduino where this is triggered twice, so we prevent it
```

Comment:
This is not a bug, it is triggered on every IP6-address, normally twice for local and global. I have set an ULA address in router and get three addresses. (local, global, ula). Also every night the router gets a new global prefix and this is triggert once with the new global address.
With TASMOT_SDK it shows the address that is new (the logging `Local` is wrong here.
From my log:
```
2024-09-10 10:19:56.123 I 15: [emsesp] Local IPv6=fe80::ea31:cdff:fefd:4308
2024-09-10 10:19:59.123 I 16: [emsesp] Local IPv6=2003:c2:4728:3f00:ea31:cdff:fefd:4308
2024-09-10 10:19:59.123 I 17: [emsesp] Local IPv6=fd00::ea31:cdff:fefd:4308
```

Reply:
wow, didn't know you could have multiple IP addresses. Shall we keep the change I made and only show the first one? On my system it shows twice with the same IP.

Reply:
The espressif arduino 2.xx only can show Local_IP6, but also gets the others. For IDF5 and tasmota-arduino we can show the ip that is transmitted to the GOT_IP6. You always have a local (FE80::) and a global address. In my case 
`::ea31:cdff:fefd:4308` is the device part, `fe00::` the local address prfix, `2003:c2:4728:3f00:` the global prefix and `fd00::` the ULA (Unique Local Address, fd:: is reseved for ULA).

Reply:
thanks for explaining that